spec/controllers/think_feel_do_engine/participants/task_status_controller_spec.rb

```
@@ -42,6 +42,17 @@ module Participants
             it { expect(response.status).to eq 400 }
           end
 
+          context "that specifies a non-existant task" do
+            before do
+              allow(task_status)
+                .to receive(:mark_complete)
+                .and_raise(ActiveRecord::InvalidForeignKey.new("invalid key"))
```

Can we just raise `ActiveRecord::InvalidForeignKey` or do we need to create a new one here?